src/pytest_mock_resources/container/mongo.py

```diff
@@ -40,10 +40,10 @@ def ports(self):
 
     def check_fn(self):
         try:
-            client = pymongo.MongoClient(self.host, self.port)
+            client = pymongo.MongoClient(self.host, self.port, timeoutMS=500)
             db = client[self.root_database]
             db.command("ismaster")
-        except pymongo.errors.ConnectionFailure:
+        except (pymongo.errors.ConnectionFailure, pymongo.errors.PyMongoError):
```

Comment:
PymongoError encompasses ConnectionFailure, can you remove Connection failure?

Reply:
Done 👍🏻